api/models/workshop.py

```diff
@@ -23,6 +25,10 @@ class Workshop(CapacityMixin, Base):
         WorkshopDifficulty,
         verbose_name=_("Difficulty"),
     )
+    location = models.ForeignKey(
```

Comment:
No, you are right, that something needs to be changed in the current design. I suggest linking workshop to current year. Maybe also location.

What do you suggest?